frontend/src/components/content-types/ContentTypeDialog.tsx

```diff
@@ -70,36 +92,14 @@ export const ContentTypeDialog: FC<ContentTypeDialogProps> = ({
       },
     },
   );
-  const validationRules = {
-    name: {
-      required: t("message.name_is_required"),
-    },
-  };
-  const onSubmitForm = async (params: IContentTypeAttributes) => {
+  const onSubmitForm = async (params) => {
     if (data) {
-      updateContentType({
-        id: data.id,
-        params,
-      });
+      updateContentType({ id: data.id, params });
     } else {
-      createContentType(params);
+      createContentType({ ...params, name: params.name || "" });
```

Comment:
```suggestion
      createContentType({ ...params, name: params.name || "" });
```
The content type "name" attribute should not be empty. In which case are getting it empty?

Reply:
Correct fixed that :heavy_check_mark: